main/ertp/api/purse.md

```diff
@@ -36,7 +36,40 @@ quatloosPurse1 = quatloosIssuer.makeEmptyPurse();
 quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
 ```
 
-## purse.getCurrentAmount()
+## purse.getRecentAmount()
+- Returns: `{Amount}`
+
+Get an `amount` describing the current digital assets balance in the `purse`.
+Of course, the returned `amount` `value` might be different the next time you
+call `getRecentAmount()` on the same `purse` if assets have been deposited or
+withdrawn from it in-between calls. 
+
+```js
+const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+// quatloos5 is a payment with balance of 5 quatloos
+const quatloosPurse.deposit(quatloos5);
+// Returns an amount with value = 5 and brand = quatloos
+const recentBalance = quatloosPurse.getRecentAmount();
```

Comment:
`getRecentAmount` returns a promise, not the amount itself. We should either `await` it or rename `recentBalance` to `recentBalancePromise`